src/components/capture/create/Store.ts

```diff
@@ -72,8 +72,16 @@ const useCaptureCreationStore = create<CaptureCreationState>(
             setDetails: (details) => {
                 set(
                     produce((state) => {
-                        if (state.details.image !== details.data.image) {
+                        if (
+                            details.data.image.length > 0 &&
+                            state.details.data.image !== details.data.image
+                        ) {
```

Comment:
We only want to kick this stuff off if there is an image selected. Otherwise we can skip.